spec.bs

```diff
@@ -1189,41 +1192,155 @@ partial interface Navigator {
-        1. Otherwise, replace the current value of the field in the interest group with that field from the received
-          interest group update JSON. (Fields that are not specified in the update JSON will remain unmodified).
-        1. Set the "last updated" of the interest group to the current time, and set the "next update after" to
-          the update rate limit period + the current time.
+    issuing too many requests at once.
+    1. Let |ig| be a deep copy of |originalInterestGroup|.
+    1. Let |update| be the result of [=fetching resource=] with |ig|'s [=interest group/daily update url=],
+      "application/json", and |owner|.
+    1. If |update| is failure, [=iteration/continue=].
+    1. Let |parsedUpdate| be the result of [=parsing JSON bytes to a JavaScript value=], given
+      |update|.
+    1. If |parsedUpdate| is failure, [=iteration/continue=].
+    1. If |parsedUpdate| is not an [=ordered map=], [=iteration/continue=].
+    1. If |parsedUpdate["name"] exists and doesn't match |ig|'s [=interest group/name=], 
+      [=iteration/continue=].
+    1. If |parsedUpdate["owner"] exists and doesn't match |ig|'s [=interest group/owner=],
+      [=iteration/continue=].
+    1. [=map/For each=] |key| → |value| of |parsedUpdate|:
+      1. Switch on |key|:
+        <dl class=switch>
+        <dt>"`priority`"
+        <dd>
+        1. If |value| is a {{double}}, set |ig|'s [=interest group/priority=]
+          to |value|.
+        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+
+        <dt>"`enableBiddingSignalsPrioritization`"
+        <dd>
+        1. If |value| is a {{boolean}}, set |ig|'s
+          [=interest group/enable bidding signals prioritization=] to |value|.
+        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+
+        <dt>"`priorityVector`"
+        <dd>
+        1. If |value| is null or an [=ordered map=] whose [=map/keys=] are
+          [=strings=] and whose [=map/values=] are {{double}}, set |ig|'s
+          [=interest group/priority vector=] to |value|.
+        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+
+        <dt>"`prioritySignalsOverrides`"
+        <dd>
+        1. If |value| is an [=ordered map=] whose [=map/keys=] are
+          [=strings=] and whose [=map/values=] are {{double}} or null:
+          1. [=map/For each=] |pvKey| → |pvValue| of |value|:
+            1. If |pvValue| is null, [=map/remove=] |ig|'s [=interest group/priority signals overrides=][|pvKey|].
+            1. Otherwise, [=map/set=] |ig|'s [=interest group/priority signals overrides=][|pvKey|] to |pvValue|.
+        1. Otherwise, jump to the step labeled <i><a href=#abort-update>Abort update</a></i>.
+
+        <dt>"`executionMode`"
+        <dd>
+        1. If |value| is "<code>compatibility</code>" or "<code>group-by-origin</code>",

```

Comment:
nit: Just for consistency I'd probably use backticks for compat and group-by-origin here, unless there's a specific reason not to.